### PR TITLE
Add, combine, move, and hide within how-tos

### PR DESCRIPTION
**Pull request by Chr1st1anSears**
This PR adds "Upgrading the build pipeline." It combines the java dependencies rebuild documents into one. It moves all the previously mentioned docs into the security section of the how-tos. And it hides both the configuring renovatebot doc and the creating a static environment doc.

### CODE REVIEW

**Comment by Chr1st1anSears** [comment]
@psturc would you please take a look at [this page](https://pr-92--boisterous-meerkat-894982.netlify.app/documentation/main/how-to-guides/secure-your-supply-chain/proc_upgrade_build_pipeline) to verify its accuracy?

File: docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+. If you have not already installed the {ProductName} GitHub application, select Install GitHub application on the “Manage build pipelines” popup page. 
+
++
+[NOTE]
+====
+If you want to restrict the GitHub application’s access to certain repositories only, use the Only select repositories option in GitHub during the installation.
+====
+
+. Returning to the “Manage build pipelines” page in {ProductName}, for any component whose build pipeline you would like to upgrade, select *Send pull request*.  
+. Select *Edit pipeline in GitHub*.
```
Comment (psturc) [inline docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_upgrade_build_pipeline.adoc:30]: Currently the button in UI says "Merge in GitHub" - not sure if that's about to change in a future?
Screenshot 2023-05-05 at 17 48 15

**Comment by psturc** [comment]
@stuartwdouglas would you have a time to double check the JVM stuff please? I'm not 100% familiar with all the stuff described here. Thanks.

**Review by psturc — APPROVED** [review]
just one question, otherwise lgtm

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**